CMakeLists.txt

```diff
@@ -368,6 +368,10 @@ if (ENABLE_RESOLVE_NEIGH)
     set(NL_KIND 3)
     set(NL_INCLUDE_DIRS ${NL3_INCLUDE_DIRS})
     set(NL_LIBRARIES ${NL3_LIBRARIES})
+    set(NL_CFLAGS ${NL3_CFLAGS})
+    set(NL_LDFLAGS ${NL3_LDFLAGS})
+    # CMake separates ldflags with a semicolon, cannot be used in a command
+    string(REPLACE ";" " " NL_LDFLAGS "${NL_LDFLAGS}")
```

Comment:
Why is this needed here but not below for NL1 ? Isn't this just list to string conversion?

Reply:
Because nl1 only adds one flag, so cmake doesn't add a semicolon.

Reply:
That is relying a bit too much on cmake internal behavior, if the flags are cmake lists then they should always be converted.. Probably the helper function should be doing this conversion to keep with cmake convention of using lists for arguments.

Reply:
Sure, I'll change it that way